src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handleCurrentPage = (page: number) => {
+    if (page < 1 || page > totalPages) {
+      return;
+    }
+
+    onPageChange(page);
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={classNames('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
```

Comment:
Using `href="#prev"` is not ideal for accessibility and may cause issues with page navigation. Consider using `href="#"` or a button element to prevent default link behavior.